pyairtable/api/base.py

```diff
@@ -15,18 +15,27 @@ class Base(ApiAbstract):
 
     base_id: str
 
-    def __init__(self, api_key: str, base_id: str, timeout=None):
+    def __init__(
+        self,
+        api_key: str,
+        base_id: str,
+        timeout=None,
+        max_records_per_request=DEFAULT_MAX_RECORDS_PER_REQUEST,
+    ):
```

Comment:
See my reasoning above for `Api`

```suggestion
    def __init__(self, api_key: str, base_id: str, **kwargs: Any):
```
_NB, don't forget to add the import for `Any` from `typing` above_